### ISSUE TITLE
Issues compiling DiffEqParamEstim

### ISSUE THREAD

**Issue by pkpddude** [JuliaDiffEq/DiffEqParamEstim.jl#132]
Windows10
HP ZBook Laptop
JuliaPro version 1.3.1-1

using DiffEqParamEstim
[ Info: Precompiling DiffEqParamEstim [1130ab10-4a5a-5621-a13d-e4788d82bd4c]
ERROR: LoadError: Dierckx is not properly installed. Please run Pkg.build("Dierckx")
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] top-level scope at C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\packages\Dierckx\9SY4A\src\Dierckx.jl:24
 [3] include at .\boot.jl:328 [inlined]
 [4] include_relative(::Module, ::String) at .\loading.jl:1105
 [5] include(::Module, ::String) at .\Base.jl:31
 [6] top-level scope at none:2
 [7] eval at .\boot.jl:330 [inlined]
 [8] eval(::Expr) at .\client.jl:425
 [9] top-level scope at .\none:3
in expression starting at C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\packages\Dierckx\9SY4A\src\Dierckx.jl:21
ERROR: LoadError: Failed to precompile Dierckx [39dd38d3-220a-591b-8e3c-4c3a8c710a94] to C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\compiled\v1.3\Dierckx\R6rOy_nvHPk.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at .\loading.jl:1283
 [3] _require(::Base.PkgId) at .\loading.jl:1024
 [4] require(::Base.PkgId) at .\loading.jl:922
 [5] require(::Module, ::Symbol) at .\loading.jl:917
 [6] include at .\boot.jl:328 [inlined]
 [7] include_relative(::Module, ::String) at .\loading.jl:1105
 [8] include(::Module, ::String) at .\Base.jl:31
 [9] top-level scope at none:2
 [10] eval at .\boot.jl:330 [inlined]
 [11] eval(::Expr) at .\client.jl:425
 [12] top-level scope at .\none:3
in expression starting at C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\packages\DiffEqParamEstim\ODIEX\src\DiffEqParamEstim.jl:2
ERROR: Failed to precompile DiffEqParamEstim [1130ab10-4a5a-5621-a13d-e4788d82bd4c] to C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\compiled\v1.3\DiffEqParamEstim\nWq0E_nvHPk.ji.
Stacktrace:
 [1] compilecache(::Base.PkgId, ::String) at .\loading.jl:1283
 [2] _require(::Base.PkgId) at .\loading.jl:1024
 [3] require(::Base.PkgId) at .\loading.jl:922
 [4] require(::Module, ::Symbol) at .\loading.jl:917

**Comment by ChrisRackauckas** [comment]
What does `]build Dierckx` give?

**Comment by pkpddude** [comment]
(@v1.3) pkg> build Dierckx
   Building Dierckx → `C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\packages\Dierckx\9SY4A\deps\build.log`
┌ Error: Error building `Dierckx`:
│ ERROR: LoadError: LibraryProduct(nothing, ["libddierckx"], :libddierckx, "Prefix(C:\\Users\\johnson.rd.1\\.juliapro\\JuliaPro_v1.3.1-1\\packages\\Dierckx\\9SY4A\\deps\\usr)") is not satisfied, cannot generate deps.jl!
│ Stacktrace:
│  [1] error(::String) at .\error.jl:33
│  [2] #write_deps_file#175(::Bool, ::Bool, ::typeof(write_deps_file), ::String, ::Array{LibraryProduct,1}) at C:\Users\julia\AppData\Local\Julia-1.3.1\share\julia\stdlib\v1.3\BinaryProvider\src\Products.jl:419
│  [3] (::BinaryProvider.var"#kw##write_deps_file")(::NamedTuple{(:verbose,),Tuple{Bool}}, ::typeof(write_deps_file), ::String, ::Array{LibraryProduct,1}) at .\tuple.jl:0
│  [4] top-level scope at C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\packages\Dierckx\9SY4A\deps\build.jl:55
│  [5] include at .\boot.jl:328 [inlined]
│  [6] include_relative(::Module, ::String) at .\loading.jl:1105
│  [7] include(::Module, ::String) at .\Base.jl:31
│  [8] include(::String) at .\client.jl:424
│  [9] top-level scope at none:5
│ in expression starting at C:\Users\johnson.rd.1\.juliapro\JuliaPro_v1.3.1-1\packages\Dierckx\9SY4A\deps\build.jl:55
└ @ Pkg.Operations C:\Users\julia\AppData\Local\Julia-1.3.1\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:898

**Comment by ChrisRackauckas** [comment]
It looks like that could be a JuliaPro issue. I have forwarded this to the relevant folk.

**Comment by pkpddude** [comment]
I gathered that because there are no problems with Julia itself.  I want to move to JuliaPro to use the functionality available in Pumas intending to stick with one platform

**Comment by bmharsha** [comment]
This issue is occurring due to a bug in JuliaPro, we are working towards fixing this. As a workaround, please execute following statements in JuliaPro to install `Dierckx` and `DiffEqParamEstim` successfully 
```
using BinaryProvider
Core.eval(BinaryProvider, :(default_platkey = platform_key_abi(string(Sys.MACHINE, compiler_abi_str(detect_compiler_abi())))))
include(joinpath(DEPOT_PATH[1],"packages","Dierckx","9SY4A","deps","build.jl"))
using Dierckx
Pkg.build("DiffEqParamEstim")
```

**Comment by bmharsha** [comment]
This issue has been fixed in the new release of JuliaPro v1.3.1-2 , you can download this binary from JuliaComputing website or by using following link: https://auth.juliacomputing.com/downloadjuliapro/juliapro/1312/JuliaPro_v1.3.1-2_build-265.exe.

NOTE:- JuliaPro v1.3.1-2 was released only on Windows to fix this issue. This issue is not reproducible on other operating systems.

**Comment by ChrisRackauckas** [comment]
Thanks!
